docs/swagger.yaml

```diff
@@ -32,6 +32,18 @@ definitions:
         description: Token is the signed JWT token string
         type: string
     type: object
+  controllers.LoginFinalize2FAResponse:
```

Comment:
Should we exclude these generated files from git?

Reply:
yes, that's a good idea